azure-pipelines/createBuildDirectories.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# This script creates the necessary directories that are required 
```

Comment:
Please include in the description why this is necessary with links to the relevant issues you've found about the problem.